assets/js/modules/ads/components/common/AdsConversionIDTextField.js

```diff
@@ -96,7 +101,9 @@ export default function AdsConversionIDTextField( { helperText } ) {
 					)
 				}
 				outlined
-				value={ adsConversionID }
+				value={ adsConversionID
+					?.replace( 'AW-', '' )
+					?.replace( 'AW', '' ) }
```

Comment:
We could do a regex of `/AW-?/` here to avoid the multiple `replace` calls, and even do one so we only replace `AW-` when it's at the start of the ID, which I think is correct 🤔 

```suggestion
					?.replace( /^AW-?/, '' ) }
```